contributing/repository/release-channels-publishing.md

```diff
@@ -0,0 +1,25 @@
+# Release Channels and Publishing
+
+Next.js has two release channels: `stable` and `canary`.
+
+## Stable
+
+The stable release is what is installed when you `npm install next`. This channel is used by the majority of Next.js users.
+
+This channel is published at a regular cadence and follows [semantic versioning](TODO).
+
+Repository maintainers can publish a new stable version using: `pnpm publish-stable`.
+The command will ask what version to publish `major`, `minor`, or `patch`.
+
+## Canary
+
+The canary channel has to be explicitly installed by users through `npm install next@canary`.
+
+This channel is published early based on the `canary` branch. It holds all changes that are waiting to be published to the stable channel.
+
+`canary` is used to test the latest features and bugfixes on real-world applications.
+
+By installing `next@canary` from time to time you can check if your application is affected by any changes that have not been published yet.
+
+Repository maintainers can publish a new stable version using: `pnpm publish-stable`.
```

Comment:
The command should probably be `pnpm publish-canary` not `pnpm publish-stable` there.
Ref: https://github.com/vercel/next.js/blob/41d4aa04c4b2e637e24a26646b7023d6aab4d98b/package.json#L36-L37

Reply:
Addressed here: https://github.com/vercel/next.js/pull/40515/commits/21a761493ad2873e32746dfa047b452c14d604b2